content/en/real_user_monitoring/reactnative/mobile_vitals.md

```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
```

Comment:
the first image is that of the flutter summary page (widget build time is a flutter vital), can we change it to the react native page?